docs/nodes/maintain/avalanchego-config-flags.md

```diff
@@ -601,11 +601,11 @@ Weight to provide to each peer when staking is disabled. Defaults to `1`.
 
 ## Subnets
 
-### Whitelist
+### Subnet Tracking
 
-#### `--whitelisted-subnets` (string)
+#### `--track-subnets` (string)
```

Comment:
-whitelisted-subnets still works, we should mark it deprecated, however,

Reply:
I put it back but we should have a better strategy for deprecation.